### PR TITLE
[10.15] [PR 1327] doc: review migrating to kiteworks

### PR DESCRIPTION
**Pull request by mmattel**
Backport of PR #1327

### CODE REVIEW

**Comment by mmattel** [comment]
@phil-davis can you restart drone?

File: modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc [hunk 1]
```diff
@@ -113,19 +111,33 @@ To be prepared for the migration, both sides need to match the prerequisites. Pl
...
+
+* Before you install the migration app, you must install the custom `rclone` binary first. The migration app cannot be activated if the _correct_ rclone is missing. If you have installed rclone before, we recommend uninstalling and install the custom rclone binary instead. After installing the rclone binary and making it executable, check the output of:
++
+--
+[source,bash]
+----
+rclone help backends
+----
+
+If the correct binary is installed and executed, it shows a list including: `Kiteworks`.
```
Comment (phil-davis) [inline modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc:128]: This is adding a lot of "new" text that I think was deleted recently.

File: modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc [hunk 2]
```diff
@@ -141,35 +153,34 @@ External mount points are not part of the automatic migration. See the following
...
 
-* Assure sufficient quota settings in the Kiteworks user profiles.
+* Ensure sufficient quota settings in the Kiteworks user profiles.
 
 * If it is planned to integrate Kiteworks into LDAP:
 +
 --
-IMPORTANT: We recommend to have the Kiteworks PCN connected and configured to an LDAP server _before_  starting the migration. This will avoid conflicting user entries that will exist in the local database additionally to the LDAP server connected.
-// New user accounts will be created during the migration, as needed. Existing user accounts will be used.
+IMPORTANT: We recommend having the Kiteworks PCN connected and configured to an LDAP server _before_  starting the migration. This will avoid conflicting user entries that will exist in the local database additionally to the LDAP server connected.
```
Comment (phil-davis) [inline modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc:177]: This change to line 165-177 is good, but was not in my original PR.
And we don't want to lose old line 166.

File: modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc (hunk 1 above)
Comment (mmattel) [reply modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc:128]: oh man 🤦‍♂️ what a mess 😢

**Comment by mmattel** [comment]
The only thing that comes to my mind is, that I delete these backports and revert the PR in master with the wrong changes. Then I would please you to redo the post check based on master as it contains the correct text and file a new PR to fix the language issues. I am in sorry for the mess...

**Comment by mmattel** [comment]
Closing because of #1332

**Pull request closed without merging**